lib/puppet_litmus/serverspec.rb

```diff
@@ -138,14 +138,14 @@ def bolt_upload_file(source, destination, options = {})
   # @param task_name [String] The name of the task to run.
   # @param params [Hash] key : value pairs to be passed to the task.
```

Comment:
you will need to add in docs for the new parameter.

Reply:
@tphoney updated the docs.